cmd/snap-preseed/main_test.go

```diff
@@ -374,6 +375,14 @@ func (fs *Fake16Seed) ModeSnaps(mode string) ([]*seed.Snap, error) {
 	return nil, nil
 }
 
+func (fs *Fake16Seed) NumSnaps() int {
+	return 0
+}
+
+func (fs *Fake16Seed) Iter(f func(sn *seed.Snap) error) error {
+	return fs.IterErr
```

Comment:
this then should probably return either some kind of unexpected calls error or nil